bindings/python/tests/ip_filter_test.py

```diff
@@ -1,8 +1,26 @@
+import ipaddress
+from typing import List
+from typing import Tuple
 import unittest
 
 import libtorrent as lt
 
 
+# boost's address formatting is currently inconsistent between alpine linux and
+# other environments, even when boost is built from source
```

Comment:
I don't think boost defines an address format. I'm pretty sure asio is just deferring to the system API.
could we just make the test compare address objects instead of strings? I would think that would make it clearer.

Btw, is this a new issue in `master`?

Reply:
Libtorrent's API returns the inconsistently-normalized strings, we can't compare to address objects directly.

We could make `normalize()` return address objects, but that would make the comparisons more verbose due to constructing lots of objects versus just strings. I prefer to keep test comparisons succinct and readable when possible.

I haven't tested, but I don't expect it's an issue in `master`, just something that gets uncovered by the expanded test suite, and only now uncovered due to running that on alpine for the first time.